db/db_options_test.cc

```diff
@@ -1209,6 +1210,39 @@ TEST_F(DBOptionsTest, BottommostCompressionOptsWithFallbackType) {
   ASSERT_EQ(kBottommostCompressionLevel, compression_opt_used.level);
 }
 
+TEST_F(DBOptionsTest, SetWithBlobCache) {
```

Comment:
We would probably want to add a test case to `options/options_test.cc` (not `db/db_options_test.cc`) to make sure the new parameter can be parsed from its string representation etc. I don't think this one provides much in terms of coverage (please feel free to remove)

Reply:
Done. Thanks!!